hilti/runtime/include/global-state.h

```diff
@@ -97,6 +97,17 @@ inline auto globalState() {
     return createGlobalState();
 }
 
+/**
+ * Returns the current global configuration without checking if it's already
+ * initialized. This is only safe to use if the runtime is already fully
+ * initialized, and should be left to internal use only where performance
+ * matters.
+ */
+inline const GlobalState* unsafeGlobalState() {
```

Comment:
Similar argument as for `configuration::unsafeGet`, I think most users outside of `init` of `globalState` should be able to use the unsafe variant (e.g., in logging or profiler code).